symphony/lib/core/class.exceptionrenderer.php

```diff
@@ -0,0 +1,282 @@
+<?php
+
+class ExceptionRenderer
+{
+    /**
+     * The render function will take an Throwable and output a formatted message.
+     *
+     * @since Symphony 2.7.0
+     *  This function works with both Exception and Throwable
+     * @since Symphony 3.0.0
+     *  This function support both html (default) and text (cli) format
+     *
+     * @param Throwable $e
+     *  The Throwable object
+     * @return string
+     *  An formatted message string
+     */
+    public static function render($e)
```

Comment:
Can see a really nice opportunity here to tap into the format of a page and render 'json' or 'xml' etc.